src/libs/AxelarQueryAPI.ts

```diff
@@ -309,30 +341,70 @@ export class AxelarQueryAPI {
       sourceToken.decimals
     );
 
-    const executionFee = destGasFeeWei.gt(minDestGasFeeWei)
+    const excludedL1ExecutionFee = destGasFeeWei.gt(minDestGasFeeWei)
       ? srcGasFeeWei
       : srcGasFeeWei.mul(minDestGasFeeWei).div(destGasFeeWei);
 
     const actualGasMultiplier = gasMultiplier === "auto" ? executeGasMultiplier : gasMultiplier;
 
-    const executionFeeWithMultiplier =
+    const excludedL1ExecutionFeeWithMultiplier =
       actualGasMultiplier > 1
-        ? executionFee.mul(actualGasMultiplier * 10000).div(10000)
-        : executionFee;
+        ? excludedL1ExecutionFee.mul(actualGasMultiplier * 10000).div(10000)
+        : excludedL1ExecutionFee;
+
+    let l1ExecutionFee = BigNumber.from(0);
```

Comment:
for the block between 355 and 391, we can encapsulate all of that in a helper function